scarb/src/compiler/compilers/starknet_contract/validations.rs

```diff
@@ -117,3 +122,43 @@ pub fn check_allowed_libfuncs(
 
     Ok(())
 }
+
+pub fn check_sierra_size_limits(classes: &[ContractClass], ws: &Workspace<'_>) {
+    for class in classes {
+        let sierra_felts = class.sierra_program.to_vec().len();
```

Comment:
```suggestion
        let sierra_felts = class.sierra_program.len();
```
Isn't this a vec already? 🤔